### PR TITLE
Feature/39445

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: components/02-elements/password-strength/password-strength.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="password-strength {{ class }}">
-    <div class="password-strength__progress {{ strengthProgressClass }}">
+    <div>
```
Comment (Aekal) [inline components/02-elements/password-strength/password-strength.hbs]: You can remove this empty `div`

File: components/04-views/registration/registration.hbs [hunk 2]
```diff
@@ -1,48 +1,58 @@
...
-
-                {{ render '@input' email }}
-                <div class="registration__input">
-                    {{ render '@input' password }}
-                    {{ render '@password-strength' passwordStrength }}
-                </div>
-                {{ render '@input' passwordConfirmation }}
+                    <legend
+                        class="
+                            ieldset__legends registration__legend
```
Comment (Aekal) [inline components/04-views/registration/registration.hbs]: Please fix the class name or remove if it's not used

File: components/02-elements/password-strength/password-strength.hbs (hunk 1 above)
Comment (szafran89) [reply components/02-elements/password-strength/password-strength.hbs]: no, I can't. I would like to keep the same structure like in Magento alpaca theme

File: components/04-views/registration/registration.hbs (hunk 2 above)
Comment (szafran89) [reply components/04-views/registration/registration.hbs]: fixed

File: components/02-elements/password-strength/password-strength.hbs (hunk 1 above)
Comment (Aekal) [reply components/02-elements/password-strength/password-strength.hbs]: Ok

**Pull request merged**